datasets/basiskaart/basiskaart.json

```diff
@@ -0,0 +1,490 @@
+{
+  "type": "dataset",
+  "id": "basiskaart",
+  "title": "Basiskaart",
+  "description": "Gegevens uit BGT (basisregistratie grootschalige topografie) en KBK (kleinschalige basis kaart)",
+  "license": "public",
+  "status": "beschikbaar",
+  "version": "0.0.1",
+  "publisher": "OIS",
+  "owner": "Gemeente Amsterdam",
+  "authorizationGrantor": "OIS", 
+  "keywords": ["GBT", "KBK", "KBK10", "KBK50"],
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "gebouwVlak",
+      "title": "Gebouw vlak",
+      "type": "table",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["schema", "id"],
+        "display": "type",
```

Comment:
Overal is "type" het display veld. Is dit daadwerkelijk uniek?

Reply:
Ja eens, type is niet uniek, zal ik ook aanpassen